project_ux/views/project_project_views.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="project.act_project_project_2_project_task_all" model="ir.actions.act_window">
+        <field name="context">{
+            'group_by': 'stage_id',
+            'search_default_project_id': [active_id],
+            'default_project_id': active_id,
+            'search_default_is_task': 1,
+        }</field>
+    </record>
+    <record id="act_project_project_2_project_sub_task_all" model="ir.actions.act_window">
+        <field name="name">Tasks</field>
```

Comment:
I am not sure, but by the id of this action its seems like the name should be Sub Tasks, instead of Tasks

Reply:
Also seems to be that this action and the one before  `project.act_project_project_2_project_task_all` are related to `res_model = project.task` so I think they should go into the `project_task_views.xml` file